src/sentry/static/sentry/app/actionCreators/modal.jsx

```diff
@@ -76,3 +76,13 @@ export function openCommandPalette(options = {}) {
       });
     });
 }
+
+export function openRecoveryOptions(options = {}) {
+  import(/* webpackChunkName: "RecoveryOptionsModal" */ '../components/modals/recoveryOptionsModal')
+    .then(mod => mod.default)
+    .then(Modal => {
+      openModal(deps => <Modal {...deps} {...options} />, {
+        modalClassName: 'recovery-options',
```

Comment:
Doesn't look like we use the class name here

Reply:
Leaving this in case design wants it... otherwise good to remove.